tools/Makefile

```diff
@@ -340,28 +356,31 @@ CMPBYT=$(ROOTDIR)/compilerlibs/ocamlcommon.cma \
 
 $(call byte_and_opt,cmpbyt,$(CMPBYT),)
 
-CAMLTEX= $(ROOTDIR)/compilerlibs/ocamlcommon.cma \
-	$(ROOTDIR)/compilerlibs/ocamlbytecomp.cma \
-	$(ROOTDIR)/compilerlibs/ocamltoplevel.cma \
-	$(ROOTDIR)/otherlibs/str/str.cma \
-	$(ROOTDIR)/otherlibs/$(UNIXLIB)/unix.cma \
-	caml_tex.ml
+caml_tex_objects := \
```

Comment:
Why rename this macro? These are libraries and a source file, none of them are object files...